### DISCUSSION TITLE
How can select Link ?

### DISCUSSION THREAD

**Discussion by zta7** [clientIO/joint#2024] [Q&A]
![image](https://user-images.githubusercontent.com/30122066/217155031-aa0eabaa-8a08-499d-bf3b-48086cfec798.png)
![image](https://user-images.githubusercontent.com/30122066/217155514-abd761b1-fb6a-4c7e-8773-b648919d9fd8.png)

**Comment by kumilingus** [comment]
If you mean selecting links using lasso, then this feature is not supported via an option yet.

Here's an [example](https://codepen.io/jointjs/pen/LYBKgPN/2a8a09292272afb06798d9a5d53fecf8) how to achieve it though.
